tests/browsing_context/test_create.py

```diff
@@ -297,14 +300,41 @@ async def test_browsingContext_create_withUserGesture_eventsEmitted(
             'type': 'event',
             'method': 'browsingContext.contextCreated',
             'params': {
-                'context': ANY_STR & Not(context_id),
+                'context': 'stable_0',
                 'url': 'about:blank',
                 'clientWindow': ANY_STR,
                 'children': None,
                 'parent': None,
                 'userContext': 'default',
                 'originalOpener': ANY_STR,
             }
+        }, {
+            'type': 'event',
+            'method': 'browsingContext.domContentLoaded',
+            'params': {
+                'context': 'stable_0',
+                'navigation': 'stable_1',
+                'timestamp': ANY_TIMESTAMP,
+                'url': url_example,
+            },
+        }, {
+            'type': 'event',
+            'method': 'browsingContext.load',
+            'params': {
+                'context': 'stable_0',
+                'navigation': 'stable_1',
+                'timestamp': ANY_TIMESTAMP,
+                'url': url_example,
+            },
+        }, {
+            'method': 'browsingContext.navigationStarted',
+            'params': {
+                'context': 'stable_0',
```

Comment:
the order seems wrong, why is the navigationStarted after domContentLoaded?

Reply:
`read_sorted_messages` sorts messages alphabetically. We could introduce sorting-persisting method, but let's keep it out of scope of this PR

Reply:
order seems to be important for these tests. I think we should use a non-sorted message reader for all new tests where the order matters.

Reply:
> let's keep it out of scope of this PR

we could open and land a PR adding a new helper first and then use it for new tests in this PR